### ISSUE TITLE
can we replace class in program with new program file in python

### ISSUE THREAD

**Issue by MalleSrikanth** [MalleSrikanth/mask-rcnn#1]
I want to replace class in  program with new program file and import that file into my file directly 
so that i can avoid using self and __init__ in the program , can i make it , or any problem will occur ?

THIS IS FROM MASK_RCNN MODEL PROGRAM FILE 

class BatchNorm(KL.BatchNormalization):
    """Extends the Keras BatchNormalization class to allow a central place
    to make changes if needed.
    Batch normalization has a negative effect on training if batches are small
    so this layer is often frozen (via setting in Config class) and functions
    as linear layer.
    """
    def call(self, inputs, training=None):
        """
        Note about training values:
            None: Train BN layers. This is the normal mode
            False: Freeze BN layers. Good when batch size is small
            True: (don't use). Set layer in training mode even when making inferences
        """
        return super(self.__class__, self).call(inputs, training=training)

**Comment by MalleSrikanth** [comment]
@Waleed Abdulla , @MASK_RCNN CAN HELP ME
